pkg/cmd/ssh_aws.go

```diff
@@ -72,13 +72,27 @@ func sshToAWSNode(nodeName, path, user, pathSSKeypair string, sshPublicKey []byt
 	fmt.Println("(3/4) Creating bastion host")
 	a.createBastionHostInstance()
 
+	fmt.Println("Waiting SSH 22 port open")
```

Comment:
Maybe  in the future it makes sense to have a logging framework with configurable verbosity levels. 
For instance to also see verbose output of the SSH command (good for understanding what went wrong).
Just to keep in mind.

Reply:
sure np, at the moments, after fixing the ssh process display looks like this when ssh failure"  we can get brief information `closed by 3.121.114.80 port 22`  to identify the issue from local to bastion instance or bastion instance to node instance.

```
SSH gardener@3.121.114.80 => gardener@ip-10-250-23-169.eu-central-1.compute.internalConnection closed by 3.121.114.80 port 22
kex_exchange_identification: Connection closed by remote host
exit status 255
```

After investigation this time, I got some new findings. I think the root cause is from AWS EC2,  As testing, SSH services may not fully running even EC2 instance status become running from pending/Initializing..  which causes SSH command may be hung at the first time until timeout exit.  so I detecting 22 port open and retry ssh command this time. 


-Ted